src/Service/LeantimeApiService.php

```diff
@@ -286,4 +581,31 @@ private function request(string $path, string $type, string $method, array $para
 
         return null;
     }
+
+    private function getDateSpanTemporal($startDate, $endDate): string
+    {
+        $currentDate = time();
+        $startDate = strtotime($startDate);
+        $endDate = strtotime($endDate);
+        if ($startDate < $currentDate && $endDate > $currentDate) {
+            return self::PRESENT;
+        } elseif ($startDate < $currentDate && $endDate < $currentDate) {
+            return self::PAST;
+        } else {
+            return self::FUTURE;
+        }
+    }
+
+    private function tagToId($tag)
```

Comment:
What is the purpose of this function?

Reply:
In Leantime, tags do not have IDs, so i made this method to try emulating an id. It returns 9 digits, representing the string. Always the same if the string is the same. SprintReportEpic needs an ID.

Reply:
Why not make the tag the id?